src/repositories/schemas.ts

```diff
@@ -34,3 +34,9 @@ export interface Admin {
   username: string;
   passwordHash: string;
 }
+
+export interface ApiKey {
+  _id: string;
+  name: string;
```

Comment:
I guess the name is supposed to be unique. Please add it as an index field in mongoDB.
Not sure whether the key should also be unique and part of the index.

see
https://github.com/LeoTuet/backend-we-want-to-sleep/blob/519800d247c57c571baca26ce0b3a431dcb8a490/src/repositories/connectToDB.ts#L23-L30

Reply:
I think saving the name of the admin that created the key would make sense here. 
See createdBy in Ballot or Token.